webapi/src/main/java/org/sakaiproject/webapi/controllers/AnnouncementsController.java

```diff
@@ -86,7 +87,9 @@
                     return null;
                 }
             })
-            .flatMap(Collection::stream).collect(Collectors.toList());
+            .flatMap(Collection::stream)
+            .sorted(Comparator.comparingLong(AnnouncementRestBean::getDate).reversed())
```

Comment:
Can't the instructor manually order (Drag-and-drop) the announcements?!

I mean definitely yuck on the idea of manually ordering, but how are they ordered elsewhere?

Reply:
In the home site announcements tool you have sort options on each column, but I was trying to just keep it simple, really. Less columns, just the title and a link to get to it directly.

Reply:
ok fine with me